MI429/MI429.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Please create all new projets with the latest .NET not on .NET FWk

Reply:
will replace it per below suggestion